pkg/trace/stats/sublayers.go

```diff
@@ -34,6 +39,195 @@ func (v SublayerValue) GoString() string {
 	return v.String()
 }
 
+// activeSpans is a set of active spans. We could use a map of spans for that.
+// But in order to re-use arrays and reduce allocations, we use two arrays.
+// in spans, we store the active spans indexes. If span 0, 2 and 5 are active, spans = [0, 2, 5]
+// in indexes, we store the index in the array spans of each span.
+// For this case, indexes[0] = 0, indexes[2] = 1 and indexes[5] = 2
+type activeSpanSet struct {
+	// capacity of the set. Also, the set can only contain numbers in [0;capacity[
+	capacity int
```

Comment:
We don't need to store this field neither here or in the calculator. It's not used anywhere. Its value is always equal to `cap(a.spans)`